src/lib/kadm5/clnt/libkadm5clnt_mit.exports

```diff
@@ -22,6 +23,7 @@ kadm5_get_config_params
 kadm5_get_policies
 kadm5_get_policy
 kadm5_get_principal
+kadm5_get_principal_keys
```

Comment:
Whatever new name you add for this for `libkadm5srv` (see above comment about ABI stability), use that on the `libkadm5clnt` side too.